src/NuGet.Core/NuGet.Packaging/Core/PackagingException.cs

```diff
@@ -39,4 +39,4 @@ public virtual ILogMessage AsLogMessage()
             return _logMessage;
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
as much as I love files ending with a new line, you haven't made any other modifications to this file.

Reply:
reverted.